ui/src/main/webapp/src/app/components/layout/group-layout.component.ts

```diff
@@ -161,20 +161,29 @@ export class GroupLayoutComponent extends AbstractComponent implements OnInit, O
                 MigrationIssuesComponent,
                 this._routeLinkProviderService,
             ),
-            new ReportMenuItem(
-                'Technologies',
-                'fa-cubes',
-                this.applicationGroup,
-                TechnologiesReportComponent,
-                this._routeLinkProviderService,
-            ),
-            new ReportMenuItem(
-                'Dependencies',
-                'fa-code-fork',
-                this.applicationGroup,
-                DependenciesReportComponent,
-                this._routeLinkProviderService
-            )
         ];
+
+        // Hide technologies and dependencies report in production mode
+        // TODO: Use process.env.ENV !== 'production' when AOT is fixed
+        let showUnfinishedReports = false;
+
+        if (showUnfinishedReports) {
```

Comment:
We could set the production flag, and not use the AOT build.
The maven production flag and the global app mode flag do not have to be locked together.